include/zephyr/net/ethernet.h

```diff
@@ -954,7 +954,7 @@ enum ethernet_hw_caps net_eth_get_hw_capabilities(struct net_if *iface)
 {
```

Comment:
You need to adjust the commit message still, the line is too long (see suggestion below). Please also mention the compiler version.
```
include/zephyr/net/ethernet.h:957:38: error: invalid conversion \
 from 'int' to 'ethernet_hw_caps' [-fpermissive]
```